tmt/base.py

```diff
@@ -2315,7 +2315,11 @@ def import_plan(self) -> Optional['Plan']:
         # Override the plan name with the local one to ensure unique names
         node.name = self.name
         # Create the plan object, save links between both plans
-        self._imported_plan = Plan(node=node, run=self.my_run, logger=self._logger.descend())
+        self._imported_plan = Plan(
+            node=node,
+            run=self.my_run,
+            logger=self._logger.descend(extra_shift=0)
```

Comment:
This looks ok to me. Or, perhaps, we should use `self._logger` here? The `import_plan()` method is taking care of the actual fetching of the remote plan and returns a new instance. Would it be a good idea to share the logger between the original reference and the real fetched instance? @happz, any thoughts on this?

Reply:
Sharing makes sense, except some minor internal details `descend()` without an actual extra indentation should give you a logger set up in the same way as the one you already have.

Reply:
I updated the PR. `self._logger` is now used.

Reply:
Thanks!